azext_iot/_help.py

```diff
@@ -259,24 +259,38 @@
     long-summary: For more information, see
```

Comment:
Reminder to provide HISTORY.rst content to be used in release notes. We should make sure to highlight all breaking changes.